packages/astro/src/core/errors/errors-data.ts

```diff
@@ -976,7 +976,8 @@ export const RedirectWithNoLocation = {
 export const UnsupportedExternalRedirect = {
 	name: 'UnsupportedExternalRedirect',
 	title: 'Unsupported or malformed URL.',
-	message: 'An external redirect must start with http or https, and must be a valid URL.',
+	message: (from: string, to: string) => `The external redirect from "${from}" to "${to}" is invalid.`,
+	hint: 'An external redirect must start with http or https, and must be a valid URL.',
```

Comment:
I decided to move it to the hint so we can use a dynamic message, but this is still the value of the `@description` in jsdocs above